src/core/worker.js

```diff
@@ -836,6 +838,10 @@ class WorkerMessageHandler {
       handler.on("GetXFADatasets", function (data) {
         return pdfManager.ensureDoc("xfaDatasets");
       });
+      handler.on("GetXRefPrevValue", async function (data) {
+        const xref = await pdfManager.ensureDoc("xref");
+        return xref.trailer.get("Prev");
```

Comment:
Can't we use the existing helper here?
```suggestion
        return pdfManager.ensureXRef("trailer").get("Prev");
```